qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/completions/QuteCompletionsForExpression.java

```diff
@@ -565,20 +567,24 @@ private CompletableFuture<CompletionList> doCompleteExpressionForObjectPart(Comp
 		Range range = QutePositionUtility.createRange(partStart, partEnd, template);
 		CompletionList list = new CompletionList();
 
-		// Completion with namespace resolver
-		// {data:item}
-		// {inject:bean}
-		doCompleteNamespaceResolvers(namespace, template, range, completionSettings, formattingSettings, list);
+		// Don't complete if in #is or #case section
+		if (!isInCaseSection(expression)) {

```

Comment:
To avoid evaluating twice !isInCaseSection(expression), I suggest that you store in a boolean the result

`boolean inCaseSection = isInCaseSection(expression);`

and you use this boolean for the following code.